src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -2381,6 +2393,21 @@ private void HandleSum(List<QueryExpression> arguments, BlittableJsonReaderObjec
 
                 _metadata.AddWhereField(new QueryFieldName(f.FieldValue, f.IsQuoted), parameters);
             }
+
+            private string GetPathWithAliasName(string propertyPath)
```

Comment:
What does it actually do? Can you show concrete example? Will that work for fields with nesting like `Foo.Bar.Baz.Bee` ?

Reply:
If for example, you query and select some fields, e.g.: 
```
spatial.point(Foo.Bar.Baz.Been.Latitude, Foo.Bar.Baz.Bee.Longitude)
...
select Foo as AliasName
```

then this method will return
```
AliasName.Bar.Baz.Bee.Latitude
```
and
```
AliasName.Bar.Baz.Bee.Longitude
```
so that the studio will know where to look for the values in the results

Reply:
What if do `select Foo.Bar.Baz.Been.Latitude as AliasName`? Won't this method fail then?

Reply:
Note  `propertyPath.IndexOf(".");`  might return `-1` if `"."` isn't found

Reply:
refactored code - works now for any variation:
`select Foo.Bar.Baz.Bee.Latitude as AliasName`
`select Foo.Bar.Baz as AliasName`
`select Foo.Bar`
etc.